src/Workspaces/Core/Portable/Workspace/WorkspaceKind.cs

```diff
@@ -18,7 +18,7 @@ public static class WorkspaceKind
         public const string MSBuild = "MSBuildWorkspace"; // This string is specifically used to avoid a breaking change.
 
         internal const string Test = nameof(Test);
-        internal const string AnyCodeRoslynWorkspace = nameof(AnyCodeRoslynWorkspace);
+        internal const string CloudEnvironmentClientWorkspace = nameof(CloudEnvironmentClientWorkspace);
```

Comment:
Thanks for this rename -- is this going to break F# or TypeScript?

Reply:
Things still seem to work in TS and F# in both open folder and non-open folder - also the any code workspace has a slightly different name - https://devdiv.visualstudio.com/DevDiv/_git/AnyCode?path=%2Fsrc%2FMicrosoft.VisualStudio.Workspace.Extensions.Implementation.VS%2FIntellisense%2FRoslynWorkspaceShim.cs&version=GBmaster&line=62&lineEnd=63&lineStartColumn=1&lineEndColumn=1&lineStyle=plain

Reply:
Thanks for checking!